histomicstk/preprocessing/color_deconvolution/__init__.py

```diff
@@ -4,19 +4,29 @@
 """
 # make functions available at the package level using shadow imports
 # since we mostly have one function per file
+from . import stain_color_map as _stain_color_map
 from .color_convolution import color_convolution
 from .complement_stain_matrix import complement_stain_matrix
 from .sparse_color_deconvolution import sparse_color_deconvolution
 
 # must be imported after ComplementStainMatrix
 from .color_deconvolution import color_deconvolution
 
+stain_color_map = _stain_color_map.stain_color_map
+__doc__ += _stain_color_map.__doc__
```

Comment:
@jbeezley, does Sphinx transfer that across module imports, which is the situation here? Or will I need to do something like
```
#: My description
stain_color_map = _stain_color_map.stain_color_map
```
in `__init__.py` to get that to work?